kornia/feature/laf.py

```diff
@@ -291,71 +307,76 @@ def generate_patch_grid_from_normalized_LAF(img: torch.Tensor,
 
 
 def extract_patches_simple(img: torch.Tensor,
-                           LAF: torch.Tensor,
-                           PS: int = 32) -> torch.Tensor:
+                           laf: torch.Tensor,
+                           PS: int = 32,
+                           normalize_lafs_before_extraction: bool = True) -> torch.Tensor:
     """
     Extract patches defined by LAFs from image tensor.
     No smoothing applied, huge aliasing (better use extract_patches_from_pyramid)
     Args:
-        LAF: (torch.Tensor).
-        images: (torch.Tensor) images, LAFs are detected in
+        img: (torch.Tensor) images, LAFs are detected in
+        laf: (torch.Tensor).
         PS: (int) patch size, default = 32
-
+        normalize_lafs_before_extraction (bool) - if True, lafs are normalized to image size, default = True
```

Comment:
the arguments that are optional I would start stating it: `normalize_lafs_before_extraction (bool, optional)`

Reply:
Disagree here, agree in other place. It is more like default value that optional - I don`t want to accept "None" here.

Reply:
would be great to somehow mark in the docs the parameters that are not strictly required